types/src/on_chain_config/aptos_features.rs

```diff
@@ -163,7 +163,6 @@ impl FeatureFlag {
             FeatureFlag::COIN_TO_FUNGIBLE_ASSET_MIGRATION,
             FeatureFlag::OBJECT_NATIVE_DERIVED_ADDRESS,
             FeatureFlag::DISPATCHABLE_FUNGIBLE_ASSET,
-            FeatureFlag::REMOVE_DETAILED_ERROR_FROM_HASH,
```

Comment:
can we also remove `is_remove_detailed_error_from_hash_enabled`? Also we can rename the feature flag to indicate this is unused?

Reply:
we probably shouldn't remove feature and thus its corresponding features. I added a comment to indicate this feature is not used.

Reply:
nit: I think you can just call the feature flag as `_DEPRECATED_...` - we already have this for transaction fees and DKG feature flags. Better than the comment because it is explicitly deprecated. I would still prefer to remove `is_remove_detailed_error_from_hash_enabled` because if feature is removed, it should not have public APIs